src/libmysqlclient-1-fixes.patch

```diff
@@ -471,3 +471,25 @@ index 1111111..2222222 100644
  /****************************************************************************
  ** Replacements for localtime_r and gmtime_r
  ****************************************************************************/
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mark Brand <mabrand@mabrand.nl>
+Date: Mon, 17 Sep 2018 11:04:30 +0200
+Subject: [PATCH 12/12] mingw-w64 6 fix
+
+
+diff --git a/mysys/my_gethwaddr.c b/mysys/my_gethwaddr.c
+index 1111111..2222222 100644
+--- a/mysys/my_gethwaddr.c
++++ b/mysys/my_gethwaddr.c
+@@ -138,8 +138,8 @@ my_bool my_gethwaddr(uchar *to)
+ #endif
+ 
+ // mprapi.h uses these undefined types
+-#define CERT_NAME_BLOB int
+-#define CRYPT_HASH_BLOB int
++//#define CERT_NAME_BLOB int
++//#define CRYPT_HASH_BLOB int
```

Comment:
Can you wrap this in `#if defined (__MINGW64_VERSION_MAJOR) && __MINGW64_VERSION_MAJOR >= 6`?

Reply:
Oops, negate that check (or test for the defs directly).